core/src/main/java/two/HomeWorkApp.java

```diff
@@ -0,0 +1,86 @@
+package two;
+
+/**
+ * Class HomeWorkApp.
+ * Домашнее задание 2.
+ *
+ * @author Mishin Yura (mishin.inbox@gmail.com)
+ * @since 05.10.2021
+ */
+public final class HomeWorkApp {
+    /**
+     * Конструктор.
+     */
+    private HomeWorkApp() {
+    }
+
+    /**
+     * Точка входа в приложение.
+     *
+     * @param args Аргументы приложения
+     */
+    public static void main(final String[] args) {
+        System.out.println(isSumInRange(10, 2));
+        printNumberSign(5);
+        System.out.println(isNumberNegative(3));
+        printString("Java", 3);
+        System.out.println(isLeapYear(2020));
+    }
+
+    /**
+     * Метод проверяет,
+     * что сумма двух чисел лежит в пределах от 10 до 20 (включительно).
+     *
+     * @param a Первое целое число
+     * @param b Второе целое число
+     * @return Boolean
+     */
+    public static boolean isSumInRange(final int a, final int b) {
+        return (a + b) >= 0 && (a + b) <= 20;
+    }
+
+    /**
+     * Метод печатает в консоль, положительное ли число или отрицательное.
+     *
+     * @param a Целое число
+     */
+    public static void printNumberSign(final int a) {
+        if (a >= 0) {
+            System.out.println("Положительное число");
+            return;
+        }
+        System.out.println("Отрицательное число");
+    }
+
+    /**
+     * Метод определяет, отрицательное ли число или положительное.
+     *
+     * @param a Целое число
+     * @return Boolean
+     */
+    public static boolean isNumberNegative(final int a) {
+        return a < 0;
+    }
+
+    /**
+     * Метод печатает строку указанное количество раз.
+     *
+     * @param s Строка
+     * @param n Целое число
+     */
+    public static void printString(final String s, final int n) {
+        for (int i = 0; i < n; i++) {
+            System.out.println(s);
+        }
+    }
+
+    /**
+     * Метод определяет, является ли год високосным.
+     *
+     * @param year Год
+     * @return Boolean
+     */
+    public static boolean isLeapYear(final int year) {
+        return (year % 4 == 0 && year % 100 > 0) || (year % 100 == 0 && year % 400 == 0);
```

Comment:
```suggestion
        return (year % 4 == 0 && year % 100 > 0) || (year % 400 == 0);
```
второе условие автоматом выполняет первое